src/grinder/views.py

```diff
@@ -1,7 +1,183 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
 
-# Create your views here.
+# Create your views here
+from django.contrib.auth.decorators import login_required
 from django.http import HttpResponse
+import requests
+import os
+import base64
+import json
+from urllib.parse import quote
+from .forms import CustomUserCreationForm
+from django.contrib.auth import authenticate, login
+from django.contrib.auth import get_user_model
+User = get_user_model()
 
-def index(request):
-    return render(request, 'base.html')
\ No newline at end of file
+print('''"Real knowledge is to know the extent of one's ignorance."'''+'\n                       '+"~Confucius~")
+
+#  Client Keys
+CLIENT_ID= os.environ.get("CLIENT_ID")
+CLIENT_SECRET= os.environ.get("CLIENT_SECRET")
+
+# Spotify URLS
+SPOTIFY_AUTH_URL = "https://accounts.spotify.com/authorize"
+SPOTIFY_TOKEN_URL = "https://accounts.spotify.com/api/token"
+SPOTIFY_API_BASE_URL = "https://api.spotify.com"
+API_VERSION = "v1"
+SPOTIFY_API_URL = "{}/{}".format(SPOTIFY_API_BASE_URL, API_VERSION)
+
+# Server-side Parameters
+CLIENT_SIDE_URL =  os.environ.get("client_side_url")
+PORT = os.environ.get("port")
+REDIRECT_URI = "{}:{}/callback/q".format(CLIENT_SIDE_URL, PORT)
+SCOPE = "playlist-modify-public playlist-modify-private user-top-read playlist-read-collaborative playlist-read-private"
+STATE = ""
+SHOW_DIALOG_bool = True
+SHOW_DIALOG_str = str(SHOW_DIALOG_bool).lower()
```

Comment:
I believe all these should be moved to config/ENV file. Probably something like `.grider`
Except for `SCOPE` I think we should keep it here, as we might need to dynamically change the scope we require.

Reply:
Basically, configurations that will be altered in this file otherwise in `src/my_django/settings.py`.